desktop-widgets/diveplanner.cpp

```diff
@@ -19,6 +19,11 @@
 #include <QMessageBox>
 #include <QSettings>
 #include <QShortcut>
+#ifndef NO_PRINTING
```

Comment:
at some point, someone will have enough free time to decide it is worth undoing this abomination.
`#ifndef NO_PRINTING` really, really, REALLY should be `#if PRINTING`
🤦🏼‍♂️